src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -97,6 +97,11 @@ export class WizardModal extends Modal {
 
 		messageChangeHandler(this._wizard.message);
 		this._wizard.onMessageChange(message => messageChangeHandler(message));
+
+		this._wizard.onLoadingChange((loading) => {
+			this.spinner = loading.loadingState;
+			this._modalOptions.spinnerTitle = loading.loadingMessage;
```

Comment:
I am assuming that the extensions will provide a different loading message for every time they are setting the loading to true. I can have a differen event for loading message. However, I felt that it is better to combine them.

Reply:
Maybe, but we also shouldn't assume that. 

Anyways you would still be getting the latest message - you'd just get it from the wizard directly instead of the event. Is there any reason you think you need to get it from the event itself?